go-fuzz-build/cover.go

```diff
@@ -670,23 +687,25 @@ func (f *File) addImport(path, name, anyIdent string) {
 	// Now refer to the package, just in case it ends up unused.
 	// That is, append to the end of the file the declaration
 	//	var _ = _cover_atomic_.AddUint32
-	reference := &ast.GenDecl{
-		Tok: token.VAR,
-		Specs: []ast.Spec{
-			&ast.ValueSpec{
-				Names: []*ast.Ident{
-					ast.NewIdent("_"),
-				},
-				Values: []ast.Expr{
-					&ast.SelectorExpr{
-						X:   ast.NewIdent(name),
-						Sel: ast.NewIdent(anyIdent),
+	if len(name) > 0 && len(anyIdent) > 0 {
```

Comment:
Out of curiosity: did we have a case when this check did not pass?

Reply:
Regarding the imports we are adding to fix this issue, we don't want to give the import a name, as the fully qualified name will be used in the instrumentation (e.g.` fs.FileMode`). Additionally, this would add an artificial usage in the instrumentation `var _ = _go_fuzz_dep_.CoverTab` to add a usage for the import. However, we don't want to add that usage, instead we remove the import with goimports. I'll add a comment to explain the reason for this check